Cargo.toml

```
@@ -16,7 +16,7 @@ edition = "2021"
 license = "MIT/Apache-2.0"
 homepage = "https://github.com/Devolutions/IronRDP"
 repository = "https://github.com/Devolutions/IronRDP"
-authors = ["Devolutions Inc. <infos@devolutions.net>"]
+authors = ["Devolutions Inc. <infos@devolutions.net>" "Teleport <goteleport.com>"]
```

Values are separated by comma:
```suggestion
authors = [
  "Devolutions Inc. <infos@devolutions.net>",
  "Teleport <goteleport.com>",
]
```

---

15ce485f09700dab2cf9928768767fb54e007b82